translations/ru.po

```diff
@@ -87,7 +87,7 @@ msgstr "Подвинуть вниз"
 
 #: src/ui_elements/palette_config.gd
 msgid "Copy as XML"
-msgstr "Скопировать как XML"
+msgstr "Скопировать XML"
```

Comment:
Are you sure about this? The context for this is, you're converting a ColorPalette resource into XML and copying that, so you can easily share it around. There's no XML until you "copy as XML".